app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreenState.kt

```diff
@@ -0,0 +1,78 @@
+package com.wire.android.ui.home.conversations
+
+import android.content.Context
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.ModalBottomSheetState
+import androidx.compose.material.ModalBottomSheetValue
+import androidx.compose.material.rememberModalBottomSheetState
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.derivedStateOf
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.platform.ClipboardManager
+import androidx.compose.ui.platform.LocalClipboardManager
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.AnnotatedString
+import com.wire.android.R
+import com.wire.android.ui.home.conversations.model.Message
+import com.wire.android.ui.home.conversations.model.MessageContent
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun rememberConversationScreenState(
+    context: Context = LocalContext.current,
```

Comment:
I'm not having the full context on the usage of this `rememberConversationScreenState` component or maybe just missing something here, but what/where would be the cases in which we'll be passing a parent context ? Would it make sense to delete this parameter and just declare it inside this composable ?

Reply:
totally makes sense